dali/operators/image/crop/crop_attr.h

```diff
@@ -34,9 +34,8 @@ namespace dali {
 class CropAttr {
  public:
   static constexpr int kNoCrop = -1;
-  explicit inline CropAttr(const OpSpec &spec)
-    : spec__(spec)
-    , batch_size__(spec__.GetArgument<int>("batch_size")) {
+  explicit inline CropAttr(const OpSpec& spec) : spec__(spec) {
+    auto max_batch_size = spec__.GetArgument<int>("max_batch_size");
```

Comment:
Maybe you can make it a member and obtain only once and reuse in `ProcessArguments`.